src/oprf.ts

```diff
@@ -129,6 +134,10 @@ export abstract class Oprf {
         this.mode = Oprf.validateMode(mode)
     }
 
+    protected getDLEQParams(): DLEQParams {
+        return { gg: this.gg, hashID: this.hashID, hash: Oprf.Crypto.hash, dst: '' }
```

Comment:
```suggestion
        return Oprf.getDLEQParams(this.ID)
```

Reply:
other option is to not declare this method and do the same as `getGroup`, which always refers to the static method

Reply:
sure, avoids duplicated code.